.env.example

```diff
@@ -4,6 +4,7 @@ EXPENSIFY_URL_COM=https://www.expensify.com.dev/
 EXPENSIFY_PARTNER_NAME=chat-expensify-com
 EXPENSIFY_PARTNER_PASSWORD=e21965746fd75f82bb66
 PUSHER_APP_KEY=ac6d22b891daae55283a
+SECURE_NGROK_URL=https://secure-expensify-user.ngrok.io/
```

Comment:
Curious, can't we reuse the `EXPENSIFY_URL_SECURE` config for this, and make people enter their secure-ngrok URL there?

Reply:
I think we could, but this way means if you want to switch from using ngrok to not using it you can just set `USE_NGROK` to false without having to change the urls often.